src/devices/Bno055/Bno055Sensor.cs

```diff
@@ -0,0 +1,486 @@
+            _i2cDevice.Write(outArray);
+            SetConfigMode(false);
+        }
+
+        /// <summary>
+        /// Set the Axis map
+        /// </summary>
+        /// <param name="x">X axis setting</param>
+        /// <param name="y">Y axis setting</param>
+        /// <param name="z">Z axis setting</param>
+        public void SetAxisMap(AxisSetting x, AxisSetting y, AxisSetting z)
+        {
+            SetConfigMode(true);
+            WriteReg(Registers.AXIS_MAP_CONFIG, (byte)(((byte)z.Axis << 4) | ((byte)y.Axis << 2) | (byte)x.Axis));
+            WriteReg(Registers.AXIS_MAP_SIGN, (byte)(((byte)z.Sign << 2) | ((byte)y.Sign << 1) | (byte)x.Sign));
+            SetConfigMode(false);
+        }
+
+        /// <summary>
+        /// Get the Axis map
+        /// </summary>
+        /// <returns>Returns an array where first element is axis X, then Y then Z</returns>
+        public AxisSetting[] GetAxisMap()
+        {
+            SetConfigMode(true);
+            var retMap = ReadByte(Registers.AXIS_MAP_CONFIG);
+            var retSign = ReadByte(Registers.AXIS_MAP_SIGN);
+            AxisSetting[] axisSettings = new AxisSetting[3];
+            axisSettings[0].Axis = (AxisMap)(retMap & 0x03);
+            axisSettings[1].Axis = (AxisMap)((retMap >> 2) & 0x03);
+            axisSettings[2].Axis = (AxisMap)((retMap >> 4) & 0x03);
+            axisSettings[0].Sign = (AxisSign)(retSign & 0x01);
+            axisSettings[1].Sign = (AxisSign)((retSign >> 1) & 0x01);
+            axisSettings[2].Sign = (AxisSign)((retSign >> 2) & 0x01);
+            SetConfigMode(false);
+            return axisSettings;
+        }
+
+        /// <summary>
+        /// Get the orientation (Eular Angles) X = Heading, Y = Roll, Z = Pitch
+        /// </summary>
+        public Vector3 Orientation
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.EULER_H_LSB);
+                // If unit is MeterG, then divide by 900, otherwise divide by 16
+                if ((_units & Units.EulerAnglesRadians) == Units.EulerAnglesRadians)
+                    return retVect / 900;
+                else
+                    return retVect / 16;
+            }
+        }
+        /// <summary>
+        /// Get the Magnetometer
+        /// </summary>
+        public Vector3 Magnetometer => GetVectorData(Registers.MAG_DATA_X_LSB) / 16;
+
+        /// <summary>
+        /// Get the gyroscope
+        /// </summary>
+        public Vector3 Gyroscope
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.GYRO_DATA_X_LSB);
+                if ((_units & Units.AngularRateRotationPerSecond) == Units.AngularRateRotationPerSecond)
+                    return retVect / 900;
+                else
+                    return retVect / 16;
+            }
+        }
+
+        /// <summary>
+        /// Get the accelerometer
+        /// </summary>
+        public Vector3 Accelerometer
```

Comment:
What's the difference between this acceleration and the LinearAcceleration?

Reply:
I explained all in the documentation. I'll copy/paste in the code as well